resources/META-INF/persistence.xml

```diff
@@ -19,7 +19,26 @@
 		    <property name="show_sql" value="true" />
 		    <!-- Scan for annotated classes and Hibernate mapping XML files -->
 		    <property name="hibernate.archive.autodetection" value="class, hbm"/>
-		    <property name="hibernate.hbm2ddl.auto" value="create" />
     	</properties>
     </persistence-unit>
+	<persistence-unit name="seed">
+		<provider>org.hibernate.jpa.HibernatePersistenceProvider</provider>

```

Comment:
@epielemeier I added another Persistence unit that is only used for seed to drop all tables and init the database.  With the auto create flag in my-pu it was dropping on every startup.  This solution allows us to drop all and create new tables at time of seed and start up correctly otherwise.